test/config/config.test.js

```diff
@@ -0,0 +1,156 @@
+/*
+ * Copyright Elasticsearch B.V. and other contributors where applicable.
+ * Licensed under the BSD 2-Clause License; you may not use this file except in
+ * compliance with the BSD 2-Clause License.
+ */
+
+'use strict'
+
+var fs = require('fs')
+var os = require('os')
+var path = require('path')
+
+var mkdirp = require('mkdirp')
+var rimraf = require('rimraf')
+const test = require('tape')
+
+const AGENT_VERSION = require('../../package.json').version
+
+const Agent = require('../../lib/agent')
+const { createMockLogger } = require('../_mock_logger')
+const { NoopApmClient } = require('../../lib/apm-client/noop-apm-client')
+const { REDACTED } = require('../../lib/constants')
+
+test('#loggingPreamble', function (t) {
```

Comment:
Perhaps:

```suggestion
test('logging preamble', function (t) {
```

The intent of the "#foo" naming with the hash sign was to indicate it was testing the "foo" property of a class/object. E.g. `test('#getServiceName()'` in "agent.test.js" was to test the `getServiceName` method of the `Agent` class.